src/components/frame/LandingPage/LandingPage.tsx

```diff
@@ -1,125 +1,169 @@
+    defaultMessage: 'Get Started',
+  },
+  createUserProfileTitle: {
+    id: `${displayName}.createUserProfile`,
+    defaultMessage: 'Create a profile',
+  },
+  createUserProfileButtonText: {
+    id: `${displayName}.createUserProfileButtonText`,
+    defaultMessage: 'Create',
+  },
+  viewUserProfileTitle: {
+    id: `${displayName}.viewUserProfile`,
+    defaultMessage: 'View profile',
   },
-  createColony: {
-    id: `${displayName}.createColony`,
-    defaultMessage: 'Create a colony',
+  viewUserProfileDescription: {
+    id: `${displayName}.viewUserProfileDescription`,
+    defaultMessage:
+      'Define your identity, track your contributions, build your reputation.',
   },
-  exploreColony: {
-    id: `${displayName}.exploreColony`,
-    defaultMessage: 'Explore the {colonyName}',
+  viewUserProfileButtonText: {
+    id: `${displayName}.viewUserProfileButtonText`,
+    defaultMessage: 'View',
   },
-  createUsername: {
-    id: `${displayName}.createUsername`,
-    defaultMessage: 'Create a username',
+  exploreMetacolonyTitle: {
+    id: `${displayName}.exploreMetacolony`,
+    defaultMessage: 'Explore the Metacolony',
+  },
+  exploreMetacolonyDescription: {
+    id: `${displayName}.exploreMetacolonyDescription`,
+    defaultMessage: 'The Colony using Colony to build Colony.',
+  },
+  exploreMetacolonyButtonText: {
+    id: `${displayName}.exploreMetacolonyButtonText`,
+    defaultMessage: 'Explore',
+  },
+  inviteBlockTitle: {
+    id: `${displayName}.inviteBlockTitle`,
+    defaultMessage: 'Invite 1 person to create a Colony',
+  },
+  inviteBlockDescription: {
+    id: `${displayName}.inviteBlockDescription`,
+    defaultMessage:
+      'You can invite only one member to create a colony of their own using the new app during the private beta with this custom invite link: app.colony.io/createcolony/{invitationCode}',
   },
 });
 
-interface LandingItemProps {
-  to: string;
-  message: MessageDescriptor;
-  onClick?: () => void;
-}
-
-const LandingItem = ({ to, message, onClick }: LandingItemProps) => (
-  <li className={styles.item}>
-    <NavLink to={to} onClick={onClick} className={styles.itemLink}>
-      <Icon className={styles.itemIcon} name="circle-plus" title={message} />
-      <span className={styles.itemTitle}>
-        <FormattedMessage {...message} />
-      </span>
-    </NavLink>
-  </li>
-);
-
 const LandingPage = () => {
-  /*
-   * Are the network contract deployed to the chain the user is connected
-   * so that they can create a new colony on it
-   */
-  const { wallet, updateUser, user, userLoading } = useAppContext();
-  const canInteractWithNetwork = useCanInteractWithNetwork();
-  const { data, loading } = useGetMetacolonyQuery();
-
-  const [metacolony] = data?.getColonyByType?.items || [];
+  const [, setHoveredItem] = useState<number>(1);
+  const navigate = useNavigate();
+  const { user, connectWallet, wallet, userLoading } = useAppContext();
+  const onUserLogin = () => {
```

Comment:
Is this right here on the landing page? Shouldn't it be somewhere higher up the route tree?

Reply:
Okay, maybe I didn't think this through. All is ok!